spec/requests/api_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe "old api endpoint", :type => :request do
+  let!(:site) { create(:site, path: 'en', locale: 'en', is_mirrored: true) }
+  let!(:article_layout) { create(:layout, :article, site: site, identifier: 'article') }
+  let!(:article) { create(:page, site: site, layout: article_layout) }
+
+  it "returns the article at /en/articles/article.json" do
+    get "/en/articles/#{article.slug}.json"
+    expect(JSON.parse(response.body)['full_path']).to eql("/en/articles/#{article.slug}")
```

Comment:
This request spec as a quick sanity check to ensure the new and old endpoints continue to work.

The detailed controller spec is still available at https://github.com/moneyadviceservice/cms/blob/master/spec/controllers/api/content_controller_spec.rb